routes/signout.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+
+router.post("/", (req, res) => {
```

Comment:
Any specific reason to use POST here instead of GET?

Reply:
since cookies are required for authentications and authorizations, so chose POST rather than GET.

Reply:
But cookies are sent with all kinds of requests : GET, POST, DELETE, PATCH. Right?

Reply:
Yes, I agree with that.

Reply:
As per the latest commit, I resolved this issue please check it and let me know if it can be resolved